### ISSUE TITLE
[A11y] Back To Top improvement request

### ISSUE THREAD

**Issue by mendeza** [ITS-HCD/nysds#1467]
# Overview

* Back To Top benefits only sighted mouse-users. It is non-operational for all other user personas. Specifically, actual focus is not moved, which makes this a focus order violation of [WCAG SC 2.4.3](https://www.w3.org/WAI/WCAG22/quickref/?showtechniques=243#focus-order).
  * **Request**: programmatically move focus to `document.body`. Why `body`? We cannot assume what every user will consider an ideal "top." For example, return to `main` may frustrate users seeking content above it. Return to `body` places the keyboard or SR user a single keystroke away from the page's Skip Link.
